mw-webapp/src/component/image/Image.cy.tsx

```diff
@@ -0,0 +1,36 @@
+import logo from "src/assets/mastersWayLogo.svg";
+import {LOGO_TEXT} from "src/component/header/Header";
+import {Image} from "src/component/image/Image";
+import {getDataCy} from "src/utils/cyTesting/getDataCy";
+
+const IMAGE_CY = "image";
+
+describe("Image component", () => {
+
+  beforeEach(() => {
+    cy.mount(
+      <Image
+        dataCy={IMAGE_CY}
+        src={logo}
+        alt={LOGO_TEXT}
+      />,
+    );
+  });
+
+  it("should render image component correctly", () => {
+    cy.get(getDataCy(IMAGE_CY)).should("exist");
+  });
+
+  it("should display a default image if src is not provided", () => {
+    cy.get(getDataCy(IMAGE_CY)).should("have.attr", "src").and("not.be.empty");
```

Comment:
Please explain what is default image should render if src is not provided? Is it our image?

Reply:
it's not a correxct test, i want to create test like "displays the alt text if the image is not displayed", removed it, anr create a new test "displays the alt text if the image is not displayed